### PR TITLE
MOB-550 Unopened Positions views

### PR DESCRIPTION
**Pull request by ruixhuang**
Screenshot_1718334191
Screenshot_1718334183

### CODE REVIEW

**Comment by mike-dydx** [comment]
@ruixhuang will wait for [UI updates](https://www.figma.com/design/mKevZOfE9nj6MZpiolKYW1/dYdX-%E2%80%BA-Mobile?node-id=5962-23399&t=dJ8LFIC75xpXw7If-4) as mentioned in DM before reviewing

**Review by mike-dydx — CHANGES_REQUESTED** [review]
@ruixhuang will wait for [UI updates](https://www.figma.com/design/mKevZOfE9nj6MZpiolKYW1/dYdX-%E2%80%BA-Mobile?node-id=5962-23399&t=dJ8LFIC75xpXw7If-4) as mentioned in DM before reviewing

**Comment by ruixhuang** [comment]
> @ruixhuang will wait for [UI updates](https://www.figma.com/design/mKevZOfE9nj6MZpiolKYW1/dYdX-%E2%80%BA-Mobile?node-id=5962-23399&t=dJ8LFIC75xpXw7If-4) as mentioned in DM before reviewing

Updated

File: v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/pendingpositions/DydxPortfolioPendingPositionItemView.kt [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                            .themeColor(ThemeColor.SemanticColor.text_tertiary),
+                    )
+
+                    Spacer(modifier = Modifier.weight(1f))
+
+                    Text(
+                        text = state.margin ?: "",
+                        style = TextStyle.dydxDefault
+                            .themeFont(fontSize = ThemeFont.FontSize.small)
+                            .themeColor(ThemeColor.SemanticColor.text_primary),
```
Comment (mike-dydx) [inline v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/pendingpositions/DydxPortfolioPendingPositionItemView.kt:124]: non-blocking: i think this is secondary in the design

**Review by mike-dydx — APPROVED** [review]

File: v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/pendingpositions/DydxPortfolioPendingPositionItemView.kt (hunk 1 above)
Comment (ruixhuang) [reply v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/pendingpositions/DydxPortfolioPendingPositionItemView.kt:124]: OK.. will update in the next PR

**Pull request merged**